### DISCUSSION TITLE
copilot not run in vscode

### DISCUSSION THREAD

**Discussion by fawdlstty** [community/community#10604] [Copilot Conversations]
I am using win11 system and cannot run copilot in vscode. The following output:
```
[ERROR] [default] [2022-01-27T02:24:26.992Z] GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"
```
What domain names does Copilot use?  I want to see if I can access it through a browser

**Comment by fawdlstty** [comment]
I found that the problem was caused by enabling the Github accelerator agent.  In the case of configuring the proxy, how do you modify the Settings?https://github.com/dotnetcore/FastGithub/issues/76

**Comment by renatodru** [comment]
The problem is about trust in chain certificate.
The solution is:
1 - install the extension in VSC "win-ca"
2 - change config "Win-ca: inject (Method to provide fetched root certificates to other externsions)" to **append**.
3 - restart VSC.

**Reply by askerlee** [reply]
Doesn't work for me

**Comment by Jiangshan00001** [comment]
works for me .
thanks
